.gitignore

```diff
@@ -57,3 +57,6 @@ docs/_build/
 # Editors
 *.swp
 .vscode
+
+# Mac OS
+.DS_Store
```

Comment:
Nit, but can you add a new line to the end of files? I have my VS Code configured to always add a new line at the end of files upon saving so I don't have to think about it.

<img width="865" alt="Screen Shot 2022-01-12 at 11 08 49 AM" src="https://user-images.githubusercontent.com/247661/149205845-5cda600a-6993-4bc3-a983-cfc2fba56314.png">

Reply:
thanks for the tip